portfolio/src/main/webapp/script.js

```diff
@@ -275,3 +294,28 @@ function loadMapMarkers(map) {
     })
   });
 }
+
+google.charts.load('current', {'packages':['corechart']});
+google.charts.setOnLoadCallback(drawChart);
+
+function drawChart(chartElement) {
+  const data = new google.visualization.DataTable();
```

Comment:
Comment both Claire and I got, should be called table or something instead of data.